src/wpsmon.c

```diff
@@ -99,12 +124,7 @@ static int is_done(char *bssid, struct libwps_data *wps) {
 	int x = list_insert(bssid);
 	if(x >= 0 && x < MAX_APS) {
 		if(wps) {
-			if(seen_list[x].checksum == 0)
```

Comment:
the checksum we set wasn't only for PBC, it would also detect changes like switching to locked (or as is the case for some router models, they remove entirely the WPS fields from probes, though i didnt test that case)

Reply:
I agree. We can combine both, checksum and flag. Checksum for json and flag for regular output.